vmdb/spec/models/mixins/authentication_mixin_spec.rb

```diff
@@ -258,17 +258,17 @@ class TestClass < ActiveRecord::Base
       end
 
       it "should have valid_authentication" do
-        @ems.authentication_valid?.should be_true
+        @ems.has_credentials?.should be_true
       end
 
       it "should have invalid authentication if userid nil" do
```

Comment:
I think these it's should have their text updated to be as clear as the new method names.